ocaml/idl/datamodel_repository.ml

```diff
@@ -94,6 +95,30 @@ let introduce_bundle =
     ~allowed_roles:(_R_POOL_OP ++ _R_CLIENT_CERT)
     ()
 
+let introduce_remote_pool =
+  call ~name:"introduce_remote_pool" ~in_oss_since:None ~lifecycle:[]
+    ~doc:
+      "Add the configuration for a new remote pool repository (note updates \
+       can be synced from it only after the remote pool has synced updates and \
+       applied the updates)"
+    ~params:
+      [
+        (String, "name_label", "The name of the repository")
+      ; (String, "name_description", "The description of the repository")
+      ; ( String
+        , "binary_url"
+        , "Base URL of binary packages of applied updates in this remote pool \
```

Comment:
I would still like an example as part of the documentation, like `https://example.com/....` that demonstrates how typcally this URL looks like. This will answer a lot more questions than just words.